content/cpp/concepts/math-functions/terms/round/round.md

```diff
@@ -0,0 +1,69 @@
+---
+Title: 'round()'
+Description: 'Returns the integer that is nearest to the argument, with halfway cases rounded away from zero.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Functions'
+  - 'Arithmetic'
+  - 'Methods'
+CatalogContent:
+  - 'learn-c-plus-plus'
+  - 'paths/computer-science'
+---
+
+The **`round()`** function returns the integer that is closest to the argument, with halfway cases rounded away from zero.
+
+## Syntax
+
+```pseudo
+round(num);
+```
+
+The `num` parameter must be a `double`,`float`, or `long double`. The return value will be an integer.
+In `halfway case` the number get round-off to the next closest integer.
+
+## Example 1
+
+```cpp
+double num = 4.5;
+double result;
+
+result = std::round(num);
+
+std::cout << "The result is " << result << "!\n";
+// Output: The result is 5!
+
+```
+
+
+## Example 2
+
+```cpp
+double num = 9.23;
+double result;
+
+result = std::round(num);
+
+std::cout << "The result is " << result << "!\n";
+// Output: The result is 9!
```

Comment:
```suggestion
#include <iostream>
#include <cmath>

int main() {
  double num1 = 9.23;
  double result1;
  result1 = std::round(num1);

  std::cout << "The result of round(9.23) is " << result1 << "!\n";

  double num2 = 4.5;
  double result2;
  result2 = std::round(num2);

  std::cout << "The result of round(4.5) is " << result2 << "!\n";
}
```

Reply:
I think we should include the `#includes` statements along with the code being nested in a `main()` method so that if a reader copy/pastes this example they can run it. 😄